nbconvert/exporters/tests/test_templateexporter.py

```diff
@@ -174,11 +174,11 @@ def test_absolute_template_dir(self):
                 f.write(test_output)
             config = Config()
             config.TemplateExporter.template_name = template
-            config.TemplateExporter.template_paths = [td]
+            config.TemplateExporter.extra_template_basedirs = [td]
```

Comment:
@MSeal is it important for compatibility that it should be `template_paths` that is assigned to?

Reply:
Unfortunately yes, but I think this was rarely directly used. I'd be ok with breaking compatibility on this field for simplicity and documenting to use the new field name if someone has an error. My understanding is that `template_name` was changed much more often and the working directory was used to set `template_paths` commonly. I might be wrong on that.